recipes/macse/build.sh

```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
-set -eu -o pipefail
+# inspired by picard build.sh
 
-outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
-mkdir -p $outdir
-mkdir -p $PREFIX/bin
-cp -R * $outdir/
-cp $RECIPE_DIR/macse.py $outdir/macse
-ls -l $outdir
-ln -s $outdir/macse $PREFIX/bin
+TGT="$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM"
+[ -d "$TGT" ] || mkdir -p "$TGT"
```

Comment:
Why not leave this as `mkdir -p "${TGT}"`? With `-p`, there won’t be an error if the directory already exists. (I realize this is how Picard’s `build.sh` does it, but it’s not needed.)